### ISSUE TITLE
Module ccc.cl Not Found Error.

### ISSUE THREAD

**Issue by debajyotin16** [ausgerechnet/cwb-ccc#63]
Hello Ausgerechnet,

I am facing this issue most of the time during "from ccc import Corpora".

home/test/cwb-ccc/ccc/cwb.py in <module>
     18 # part of module
     19 from .cache import Cache, generate_idx, generate_library_idx
---> 20 from .cl import Corpus as Attributes
     21 from .concordances import Concordance
     22 from .counts import Counts, cwb_scan_corpus

ModuleNotFoundError: No module named 'ccc.cl'

**Comment by ausgerechnet** [comment]
looks like the C-extension was not properly compiled.

did you install from source or from PyPI?

which OS are you on, which Python version are you using?

**Comment by debajyotin16** [comment]
Am I using "Python 3.9.16"
I have successfully installed "cwb-ccc" on Ubuntu 22.04 LTS. But I faced module not found this error, then I updated my system 3-4 times then it started working. 

Right now I am trying to install this package on "Google Colab" for some testing reasons.
I have downloaded the git repo and mounted it too via google drive and I use "!pip install -r requirements.txt" to install the "cwb-ccc"

Am I using "Python 3.9.16"
I have successfully installed "cwb-ccc" on Ubuntu 22.04 LTS. But I faced module not found this error, then I updated my system 3-4 times then it started working.
